tools/startup/startup-basics-only.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+readonly DOWNLOAD_DIR_NAME="downloads_$(date +%Y%m%d_%H%M%S)"
+readonly DOWNLOAD_DIR_ABS_PATH="$(pwd)/$DOWNLOAD_DIR_NAME"
+readonly BASE_ABS_PATH=${HOME}/base
+readonly BAZEL_VERSION=0.21.0
```

Comment:
We can already set this to the most recent Bazel version.

Reply:
Updated to `0.23.2`